models/account.py

```diff
@@ -70,7 +70,7 @@ def get_account(email: str = None, account_id: str = None, azure_ad_subject_id=N
         if email is account_id is azure_ad_subject_id is None:
             raise TypeError("Requires an email address, azure_ad_subject_id or account_id")
 
-        url = Config.ACCOUNT_STORE_API_HOST + Config.ACCOUNTS_ENDPOINT
+        url = Config.ACCOUNT_STORE_API_HOST + "/accounts"
```

Comment:
Wanted to update this but lots of the unit tests break as they depend on the invalid url `account_store` and appears non-trivial to fix.